### DISCUSSION TITLE
Eslint plugin

### DISCUSSION THREAD

**Discussion by Demivan** [fluent-vue/fluent-vue#743] [Ideas]
Check for:
- [ ] Missing translation in the main locale
- [ ] ~~Interpolation of the translation key~~ No computed keys
- [ ] Reusing non-global translations

**Comment by samuelstroschein** [comment]
I just opened a similar feature request in the inlang repository https://github.com/inlang/inlang/discussions/108. 
> _The inlang project should work out of the box with this package. Adding features like linting etc._

Can you elaborate on what you mean with "interpolation of translation key" and "reusing non-global translations"?

**Reply by Demivan** [reply]
> Interpolation of translation key

Disallow `t('key-' + id)`  - so no computed keys.
Port of https://eslint-plugin-vue-i18n.intlify.dev/rules/no-dynamic-keys.html#rule-details

> Reusing non-global translations

Don't allow the same translation to be used multiple times in one component.
For example having `like = Like` translation and using it for like button and for dialog title. Better to split them into multiple translations that just happen to be the same: `like-btn = Like` and `dialog-title = Like`.

**Reply by samuelstroschein** [reply]
Thanks for the clarification. 

Did you receive my mail?
